app/main.py

```diff
@@ -1 +1,63 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not isinstance(command, str):
+        print("Incorrect format")
+        return
+
+    cmd_list = command.strip().split(" ")
+
+    if len(cmd_list) and cmd_list[0] != "mv":
+        print("Command should be 'mv'")
+        return
+
+    if len(cmd_list) != 3:
+        print("Command 'mv' should contains exactly two arguments")
+        return
+
+    source_name = cmd_list[-2]
+    destination_name = cmd_list[-1]
+
+    if source_name == destination_name:
+        print("Warning: arguments are the same")
+        return
+
+    if source_name[-1] == "/":
```

Comment:
This check assumes that the source is a directory if it ends with a '/'. However, this might not be a reliable check across different operating systems. Consider using `os.path.isdir(source_name)` to check if the source is a directory.